2014-01-08T10:41:33.059844+00:00 heroku[worker.2]: Starting process with command `python worker.py`
2014-01-08T10:41:33.262691+00:00 app[web.1]: 2014-01-08 10:41:33 [2] [INFO] Listening at: http://0.0.0.0:22067 (2)
2014-01-08T10:41:33.262781+00:00 app[web.1]: 2014-01-08 10:41:33 [2] [INFO] Using worker: sync
2014-01-08T10:41:33.262022+00:00 app[web.1]: 2014-01-08 10:41:33 [2] [INFO] Starting gunicorn 18.0
2014-01-08T10:41:33.272640+00:00 app[web.1]: 2014-01-08 10:41:33 [7] [INFO] Booting worker with pid: 7
2014-01-08T10:41:33.382077+00:00 heroku[worker.1]: Starting process with command `python worker.py`
2014-01-08T10:41:33.731198+00:00 heroku[worker.2]: State changed from starting to up
2014-01-08T10:41:33.864936+00:00 heroku[web.1]: State changed from starting to up
2014-01-08T10:41:34.027918+00:00 heroku[worker.1]: State changed from starting to up
2014-01-08T10:41:34.673344+00:00 app[worker.1]: 10:41:34 RQ worker started, version 0.3.13
2014-01-08T10:41:34.675575+00:00 app[worker.1]: 10:41:34 
2014-01-08T10:41:34.675739+00:00 app[worker.1]: 10:41:34 *** Listening on high, default, low...
2014-01-08T10:41:35.326924+00:00 app[worker.2]: 10:41:35 *** Listening on high, default, low...
2014-01-08T10:41:35.324333+00:00 app[worker.2]: 10:41:35 RQ worker started, version 0.3.13
2014-01-08T10:41:35.326612+00:00 app[worker.2]: 10:41:35 
2014-01-08T10:41:35.421869+00:00 app[web.1]: 2014-01-08 10:41:35 [7] [ERROR] Exception in worker process:
2014-01-08T10:41:35.421869+00:00 app[web.1]: Traceback (most recent call last):
2014-01-08T10:41:35.421869+00:00 app[web.1]:     worker.init_process()
2014-01-08T10:41:35.421869+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-01-08T10:41:35.421869+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-01-08T10:41:35.421869+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-01-08T10:41:35.421869+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-01-08T10:41:35.421869+00:00 app[web.1]:     self.callable = self.load()
2014-01-08T10:41:35.421869+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-01-08T10:41:35.422097+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-01-08T10:41:35.421869+00:00 app[web.1]:     return self.load_wsgiapp()
2014-01-08T10:41:35.422097+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-01-08T10:41:35.422097+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-01-08T10:41:35.422097+00:00 app[web.1]:     __import__(module)
2014-01-08T10:41:35.422097+00:00 app[web.1]:   File "/app/rosterrun.py", line 58, in <module>
2014-01-08T10:41:35.422097+00:00 app[web.1]:     job
2014-01-08T10:41:35.422097+00:00 app[web.1]: NameError: name 'job' is not defined
20
14-01-08T10:41:35.422097+00:00 app[web.1]: Traceback (most recent call last):
2014-01-08T10:41:35.422097+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-01-08T10:41:35.422097+00:00 app[web.1]:     worker.init_process()
2014-01-08T10:41:35.422274+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-01-08T10:41:35.422274+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-01-08T10:41:35.422274+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-01-08T10:41:35.422274+00:00 app[web.1]:     self.callable = self.load()
2014-01-08T10:41:35.422274+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-01-08T10:41:35.422274+00:00 app[web.1]:     return self.load_wsgiapp()
2014-01-08T10:41:35.422274+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-01-08T10:41:35.422768+00:00 app[web.1]:   File "/app/rosterrun.py", line 58, in <module>
2014-01-08T10:41:35.422274+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-01-08T10:41:35.422274+00:00 app[web.1]:     __import__(module)
2014-01-08T10:41:35.422274+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-01-08T10:41:35.422768+00:00 app[web.1]:     job
2014-01-08T10:41:35.422768+00:00 app[web.1]: NameError: name 'job' is not defined
2014-01-08T10:41:35.422768+00:00 app[web.1]: 2014-01-08 10:41:35 [7] [INFO] Worker exiting (pid: 7)
2014-01-08T10:41:35.572887+00:00 app[web.1]: 2014-01-08 10:41:35 [2] [INFO] Shutting down: Master
2014-01-08T10:41:35.572887+00:00 app[web.1]: 2014-01-08 10:41:35 [2] [INFO] Reason: Worker failed to boot.
2014-01-08T10:41:36.795988+00:00 heroku[web.1]: Process exited with status 3
2014-01-08T10:41:36.807859+00:00 heroku[web.1]: State changed from up to crashed
2014-01-08T10:41:36.808887+00:00 heroku[web.1]: State changed from crashed to starting
2014-01-08T10:41:49.089824+00:00 heroku[web.1]: Starting process with command `gunicorn rosterrun:app`
2014-01-08T10:41:50.554753+00:00 app[web.1]: 2014-01-08 10:41:50 [2] [INFO] Starting gunicorn 18.0
2014-01-08T10:41:50.567558+00:00 app[web.1]: 2014-01-08 10:41:50 [7] [INFO] Booting worker with pid: 7
2014-01-08T10:41:50.555974+00:00 app[web.1]: 2014-01-08 10:41:50 [2] [INFO] Listening at: http://0.0.0.0:50395 (2)
2014-01-08T10:41:50.556269+00:00 app[web.1]: 2014-01-08 10:41:50 [2] [INFO] Using worker: sync
2014-01-08T10:41:50.935730+00:00 heroku[web.1]: State changed from starting to up
2014-01-08T10:41:20.934239+00:00 heroku[api]: Deploy 84680a3 by andygee@gmail.com
2014-01-08T10:41:20.965631+00:00 heroku[api]: Release v127 created by andygee@gmail.com
2014-01-08T10:41:53.773609+00:00 app[web.1]: 2014-01-08 10:41:53 [7] [ERROR] Exception in worker process:
2014-01-08T10:41:53.773609+00:00 app[web.1]: Traceback (most recent call last):
2014-01-08T10:41:53.773609+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-01-08T10:41:53.773609+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-01-08T10:41:53.773609+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-01-08T10:41:53.773609+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-01-08T10:41:53.773609+00:00 app[web.1]:     self.callable = self.load()
2014-01-08T10:41:53.773609+00:00 app[web.1]:     worker.init_process()
2014-01-08T10:41:53.773609+00:00 app[web.1]:     return self.load_wsgiapp()
2014-01-08T10:41:53.773840+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-01-08T10:41:53.773840+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-01-08T10:41:53.773840+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-01-08T10:41:53.773840+00:00 app[web.1]:     __import__(module)
2014-01-08T10:41:53.773840+00:00 app[web.1]:   File "/app/rosterrun.py", line 58, in <module>
2014-01-08T10:41:53.773840+00:00 app[web.1]:     job
2014-01-08T10:41:53.773840+00:00 app[web.1]: NameError: name 'job' is not defined
2014-01-08T10:41:53.773609+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-01-08T10:41:53.774011+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-01-08T10:41:53.774011+00:00 app[web.1]:     self.callable = self.load()
2014-01-08T10:41:53.774011+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-01-08T10:41:53.774011+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-01-08T10:41:53.773840+00:00 app[web.1]: Traceback (most recent call last):
2014-01-08T10:41:53.773840+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-01-08T10:41:53.773840+00:00 app[web.1]:     worker.init_process()
2014-01-08T10:41:53.774011+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-01-08T10:41:53.774011+00:00 app[web.1]:     return self.load_wsgiapp()
2014-01-08T10:41:53.774011+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-01-08T10:41:53.774011+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-01-08T10:41:53.774011+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py",
 line 354, in import_app
2014-01-08T10:41:53.774011+00:00 app[web.1]:     __import__(module)
2014-01-08T10:41:53.774582+00:00 app[web.1]:   File "/app/rosterrun.py", line 58, in <module>
2014-01-08T10:41:53.774582+00:00 app[web.1]:     job
2014-01-08T10:41:53.774582+00:00 app[web.1]: NameError: name 'job' is not defined
2014-01-08T10:41:53.774582+00:00 app[web.1]: 2014-01-08 10:41:53 [7] [INFO] Worker exiting (pid: 7)
2014-01-08T10:41:53.932667+00:00 app[web.1]: 2014-01-08 10:41:53 [2] [INFO] Shutting down: Master
2014-01-08T10:41:53.932667+00:00 app[web.1]: 2014-01-08 10:41:53 [2] [INFO] Reason: Worker failed to boot.
2014-01-08T10:41:53.832235+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path=/favicon.ico host=rosterrun.herokuapp.com fwd="107.16.122.17" dyno=web.1 connect=28ms service=1680ms status=503 bytes=0
2014-01-08T10:41:55.254859+00:00 heroku[web.1]: Process exited with status 3
2014-01-08T10:41:55.273584+00:00 heroku[web.1]: State changed from up to crashed
2014-01-08T10:42:03.526605+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path=/ host=rosterrun.herokuapp.com fwd="107.16.122.17" dyno= connect= service= status=503 bytes=
2014-01-08T10:42:03.879715+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path=/favicon.ico host=rosterrun.herokuapp.com fwd="107.16.122.17" dyno= connect= service= status=503 bytes=
Terminate batch job (Y/N)? 

C:\Users\andy\Documents\GitHub\rosterrun>